src/SFA.DAS.Commitments.Support.SubSite/Services/CommitmentStatusCalculator.cs

```diff
@@ -0,0 +1,56 @@
+﻿using SFA.DAS.Commitments.Domain.Entities;
+using SFA.DAS.Commitments.Support.SubSite.Enums;
+
+namespace SFA.DAS.Commitments.Support.SubSite.Services
+{
+    public sealed class CommitmentStatusCalculator : ICommitmentStatusCalculator
```

Comment:
This class has been updated recently in EmployerCommitments, so this version is out of date.
Ideally, we should only have one copy of this code, as going forward it's going to be a pain to keep the copies in sync, especially as they currently exist in different repos, and is likely to missed.

Reply:
There is absolutely no way this class should have been copied!!!!  It also looks like any tests haven't been pulled across with it.